src/test/groovy/io/seqera/wave/controller/RegistryControllerLocalTest.groovy

```diff
@@ -147,22 +147,25 @@ class RegistryControllerLocalTest extends Specification implements DockerRegistr
                 h.add('Accept', it)
             }
         })
-        HttpResponse<Map> response = client.toBlocking().exchange(request, Map)
+        HttpResponse<Map> response = client.toBlocking().exchange(request, String)
```

Comment:
the error may be related to this, because now response is not mapped to map directly

Reply:
there was a body bytes field in HttpResponse of micronaut 3 
<img width="1330" alt="Screenshot 2024-06-12 at 23 49 52" src="https://github.com/seqeralabs/wave/assets/6575288/78f6e598-fdda-45f8-bf3a-ae8826178c8e">

which is not anymore present in micronaut 4 and also status field is missing
<img width="1068" alt="Screenshot 2024-06-12 at 23 50 27" src="https://github.com/seqeralabs/wave/assets/6575288/df222406-52f5-463b-9be9-3b0a0d2f9409">